bin/reth/src/node/mod.rs

```diff
@@ -822,6 +842,18 @@ impl<Ext: RethCliExt> NodeCommand<Ext> {
 
         Ok(pipeline)
     }
+
+    /// Change rpc port numbers based on the instance number.
+    ///
+    /// Port numbers are chosen in a way that it's very rare that they conflict with each other.
+    fn check_instance(&mut self) {
```

Comment:
I think `adjust_instance_ports` or something is better, `check_instance` implies we are checking if the ports are valid